core/src/popover/popover.tsx

```diff
@@ -12,9 +12,21 @@ interface ContentProps {
 }
 
 export interface PopoverProps {
+	/**
+	 * The component that will be render Popover when action on it
+	 */
 	target: (props: TargetProps) => React.ReactNode;
+	/**
+	 * The elements to be displayed within the Popover
+	 */
 	content: (props: ContentProps) => React.ReactNode;
+	/**
+	 * `target`'s wrapper
+	 */
```

Comment:
This is over-simplified, but it's ok for now. The usage should actually be changed to whether it's "block" or "inline". We won't need a whole function for this.